configure

```diff
@@ -184,10 +184,13 @@ while true; do
   fromuser=""
   if [ -z "$CUDA_TOOLKIT_PATH" ]; then
     default_cuda_path=/usr/local/cuda
+    if is_windows; then
+      default_cuda_path="C:/Program Files/NVIDIA GPU Computing Toolkit/CUDA/v8.0"
```

Comment:
The CUDA installer seems to have set `%CUDA_PATH%` to this value on my machine. Should we try that location?